book/website/reproducible-research/case-studies.md

```diff
@@ -0,0 +1,21 @@
+(rr-case-studies)=
+# Case Studies: Reproducible Research
+
+## Prerequisites / recommended skill level
+
+This chapter assumes familiarity with all topics covered in the [chapter: guide to reproducible research]. 
+The actually required skills depend on the case study since not all examples
+make use of all methods covered in the guide to reproducible research.
```

Comment:
```suggestion
Since not all examples will demonstrate the use of the same combinations of methods covered in this guide, the prerequisite and skill level will vary across the case studies.
Therefore, we recommend the authors to indicate those skills for each case study separately.
```